semantic-tags/src/AST/Element.hs

```diff
@@ -0,0 +1,82 @@
+{-# LANGUAGE AllowAmbiguousTypes, DataKinds, FlexibleInstances, MultiParamTypeClasses, PatternSynonyms, ScopedTypeVariables, TypeApplications, TypeFamilies, TypeOperators, UndecidableInstances, ViewPatterns #-}
+module AST.Element
```

Comment:
IMO this belongs elsewhere—e.g. in `semantic-ast`. However, there’s no reason to either rush #243 or to block this on it, so I’ve placed this here in the aspiration of moving it into `semantic-ast` whenever @aymannadeem is ready for it.

I feel like that’s a pretty decent strength of our new approach to organization: when it isn’t convenient for something to be in the right place, we can put it temporarily in the wrong place and fix it up in a later PR quite easily. We’ve done that already with `Language.Python.Tags`, for example!

Reply:
I agree with that move, and agree that we don’t have to do it right now.